mmdet/core/bbox/assigners/atss_cost_assigner.py

```diff
@@ -0,0 +1,200 @@
+import torch
+
+from ..builder import BBOX_ASSIGNERS
+from ..iou_calculators import build_iou_calculator
+from .assign_result import AssignResult
+from .base_assigner import BaseAssigner
+
+
+@BBOX_ASSIGNERS.register_module()
+class ATSSCostAssigner(BaseAssigner):
+    """Assign a corresponding gt bbox or background to each bbox.
```

Comment:
It is best to add the difference between note description and atss assigner here